### PR TITLE
Add code owner for config file

### PR DESCRIPTION
**Pull request by zhouqiang-cl**
Signed-off-by: Qiang Zhou <zhouqiang.cl@gmail.com>

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

```release-note
Add code owner
```

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Require review from domain experts when the PR modified significant config files.
+/tests/integrations/config/test-custom.toml @tikv/tikv-configuration-reviewer
```
Comment (BusyJay) [inline .github/CODEOWNERS:2]: This is only for testing, doesn't need to be reviewed.

Comment (zhouqiang-cl) [reply .github/CODEOWNERS]: Fixed

**Review by BusyJay — APPROVED** [review]

**Comment by zhouqiang-cl** [comment]
@hicqu PTAL

**Review by hicqu — APPROVED** [review]

**Pull request merged**